### PR TITLE
Updated the WAST version

### PR DESCRIPTION
**Pull request by wycst**
Updated the WAST version to optimize the read and write performance of double

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ The results here-below were computed on January the 30th, 2024 with the followin
 | tapestry     | 5.8.3    |
 | underscore   | 1.97     | 
 | yasson       | 3.0.3    |
-| wast         | 0.0.12.1 |
+| wast         | 0.0.13.2 |
```
Comment (fabienrenaud) [inline README.md:86]: Please revert this change in the readme as this list of versions is meant to indicate what lib versions were tested on the charts on this page.

File: src/main/java/com/github/fabienrenaud/jjb/databind/Deserialization.java [hunk 2]
```diff
@@ -153,6 +153,6 @@ public Object quickbuf_json() throws Exception {
     @Benchmark
     @Override
     public Object wast() throws Exception {
-        return io.github.wycst.wast.json.JSON.parseObject(JSON_SOURCE().nextString(), JSON_SOURCE().pojoType(), ReadOption.UseJDKDoubleParser);
+        return io.github.wycst.wast.json.JSON.parseObject(JSON_SOURCE().nextString(), JSON_SOURCE().pojoType()/*, ReadOption.UseJDKDoubleParser*/);
```
Comment (fabienrenaud) [inline src/main/java/com/github/fabienrenaud/jjb/databind/Deserialization.java:156]: Either:
 - remove entirely the `ReadOption.UseJDKDoubleParse` part to show/suggest it's the normal default setup;
 - or, add a comment explaining why this is kept here but commented.

File: src/main/java/com/github/fabienrenaud/jjb/databind/Serialization.java [hunk 3]
```diff
@@ -187,7 +187,7 @@ public Object quickbuf_json() throws Exception {
     @Override
     public Object wast() throws Exception {
         ByteArrayOutputStream baos = JsonUtils.byteArrayOutputStream();
-        io.github.wycst.wast.json.JSON.writeJsonTo(JSON_SOURCE().nextPojo(), baos, WriteOption.WriteDecimalUseToString);
+        io.github.wycst.wast.json.JSON.writeJsonTo(JSON_SOURCE().nextPojo(), baos/*, WriteOption.WriteDecimalUseToString*/);
```
Comment (fabienrenaud) [inline src/main/java/com/github/fabienrenaud/jjb/databind/Serialization.java:190]: ditto

**Review by fabienrenaud — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (wycst) [reply README.md:86]: This library, which has just been added, has not been tested yet, and there is no performance data for this library in the chart. It seems that it can be updated.

File: src/main/java/com/github/fabienrenaud/jjb/databind/Serialization.java (hunk 3 above)
Comment (wycst) [reply src/main/java/com/github/fabienrenaud/jjb/databind/Serialization.java]: It has been cleared

File: README.md (hunk 1 above)
Comment (fabienrenaud) [reply README.md:86]: Fair.

**Review by fabienrenaud — APPROVED** [review]

**Comment by fabienrenaud** [comment]
Thank you!
